std/range/package.d

```diff
@@ -9865,6 +9866,15 @@ sgi.com/tech/stl/binary_search.html, binary_search).
         return !predFun(value, _input[i]);
     }
 
+/**
+Performs $(D contains) with reverse order of arguments.
```

Comment:
This is just stating implementation details. Also needs a `Params:` and `Returns:` section.